validateOffload.py

```diff
@@ -118,9 +118,9 @@ def output(self, out: TftAggregateOutput):
 
         if self.iperf_pod_type == PodType.HOSTBACKED:
             if isinstance(self._iperf_instance, IperfClient):
-                logger.info(f"The client VF representor ovn-k8s-mp0_0 does not exist")
```

Comment:
It should still be there. Did you test this with ovs HWOL?

Reply:
It no longer has the `_0` suffix